pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -396,3 +394,70 @@ type GettableGrafanaRule struct {
 	Provenance      Provenance          `json:"provenance,omitempty" yaml:"provenance,omitempty"`
 	IsPaused        bool                `json:"is_paused" yaml:"is_paused"`
 }
+
+// AlertQuery represents a single query associated with an alert definition.
+type AlertQuery struct {
```

Comment:
What about adding this into a `models.go` file? Or `input_models.go`?

Reply:
these files are mostly models + api definitions in the comments